docker/jdk8/x86_64/ubuntu/Dockerfile

```diff
@@ -31,24 +41,26 @@ RUN apt-get update \
     libcups2-dev \
     libfreetype6-dev \
     libasound2-dev \
-    openjdk-7-jdk \
+    zulu-7 \
```

Comment:
I believe this may have been because we bumped the ubuntu to 16.04, and it does not have a openjdk-7, certainly does not seem to be one at https://packages.ubuntu.com/xenial/java/

Reply:
This should work
```bash
sudo add-apt-repository ppa:openjdk-r/ppa  
sudo apt-get update   
sudo apt-get install openjdk-7-jdk
````

Reply:
Thinking about complexity of this - so much jdks to build other jdks on so many system -  and new STS x LTS relase system, this may become unmaintainable.
Maybe boot jdk do not need to be distribution's but stored  legacy AdoptOpenJDK  portable one?

Reply:
@gdams That's an unsafe PPA (it's why I swapped to Zulu)

Reply:
@judovana Agreed, although we don't have JDK <= 7 hence I defaulted to Azul's Zulu

Reply:
Well I hope AdoptOJDK will be producing binaries of JDK7 in some time.  In all cases, I have both jdk7u and icedtea 2.6 and icedtea 2.7 portable builds.   Unluckily I do not have  any jdk6.
So if jdk7  is build by adopt openjdk, it can bootstrap from itself.  
The only question is where to enter this cycle.  Is usable jdk6 still somewhere obtainable?